src/protocols/index.js

```diff
@@ -30,8 +30,25 @@
 /** @typedef {import('./lending-protocol.js').RepayOptions} RepayOptions */
 /** @typedef {import('./lending-protocol.js').RepayResult} RepayResult */
 
+/** @typedef {import('./fiat-protocol.js').BuyOptions} BuyOptions */
+/** @typedef {import('./fiat-protocol.js').BuyCommonOptions} BuyCommonOptions */
+/** @typedef {import('./fiat-protocol.js').BuyExactCryptoAmountOptions} BuyExactCryptoAmountOptions */
+/** @typedef {import('./fiat-protocol.js').BuyWithFiatAmountOptions} BuyWithFiatAmountOptions */
```

Comment:
You can export only 'BuyOptions':
```suggestion
/** @typedef {import('./fiat-protocol.js').BuyOptions} BuyOptions */
```